modules/wakedynamics/src/WakeDynamics.f90

```diff
@@ -1475,7 +1475,7 @@ subroutine WD_CalcOutput( t, u, p, x, xd, z, OtherState, y, m, errStat, errMsg )
 ! All of the calculated output channels are placed into the m%AllOuts(:), while the channels selected for outputs are
 ! placed in the y%WriteOutput(:) array.
 !..................................................................................................................................
-   use FVW_VTK ! 
+   use VTK ! 
```

Comment:
is this `use` statement needed now that VTK is in the nwtc library?

Reply:
It's not in the library yet.